modules/core/arrow-test/src/main/kotlin/arrow/test/laws/Law.kt

```diff
@@ -19,9 +22,11 @@ fun <A> A.equalUnderTheLaw(b: A, eq: Eq<A>): Boolean =
   eq.run { eqv(b) }
 
 fun <A> A.shouldBeEq(b: A, eq: Eq<A>): Unit = eq.run {
-  this.should {
-    io.kotlintest.Result(eqv(b), "Expected: $this but found: $b", "$this and $b should be equal")
-  }
+  this.should(object: Matcher<Eq<A>> {
```

Comment:
revert?

Reply:
Imho the method now does what it was originally intended for. We could discuss it with the author and then put it in its own PR, make the change here or revert it.

Reply:
The value in `override fun test(value: Eq<A>): Result {` is never used, so there's something there that doesn't add up.

Reply:
yeah right, thats not that clean. Additional changes should then be made in new PR. I will revert for now and keep this thing in mind.

Reply:
All clear now, merging!